glide.lock

```diff
@@ -233,7 +241,7 @@ imports:
 - name: gopkg.in/yaml.v2
   version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
 - name: k8s.io/api
-  version: 11147472b7c934c474a2c484af3c0c5210b7a3af
+  version: 055879e38033d1526b0fe169089a055c33fa3990
```

Comment:
K8s.io/api shouldn't have been updated by your change. 

I think there's a circular dependency here.  Maybe the cobra version shouldn't be specified in glide.yaml?

Reply:
I think that this is a "wanted" side-effect of glide. If we don't lock it to a range, or a specific version it will always be updated if possible.

Reply:
What's strange is that the version we had is also not the one which is mention in k8s/client-go on v6.0.0 (https://github.com/kubernetes/client-go/blob/78700dec6369ba22221b72770783300f143df150/Godeps/Godeps.json).

Reply:
Rebased on latest master which includes #864. glide.lock looks much better now.